bin/docker/kafka_bridge_config_generator.sh

```diff
@@ -148,11 +148,25 @@ HTTP_PROPERTIES=$(cat <<-EOF
 http.host=${KAFKA_BRIDGE_HTTP_HOST}
 http.port=${KAFKA_BRIDGE_HTTP_PORT}
 http.cors.enabled=${KAFKA_BRIDGE_CORS_ENABLED}
-http.cors.allowedOrigins=${KAFKA_BRIDGE_CORS_ALLOWED_ORIGINS}
-http.cors.allowedMethods=${KAFKA_BRIDGE_CORS_ALLOWED_METHODS}
 EOF
 )
 
+if [ "${KAFKA_BRIDGE_CORS_ALLOWED_ORIGINS}" ]; then
```

Comment:
I would add a comment on both the `if` statements highlighting that's needed because Quarkus doesn't allow empty configuration parameters.
Something like ...

"Quarkus doesn't allow empty configuration parameters if the env var is not set by the operator"